server/build/docker-compose.common.yml

```diff
@@ -51,7 +51,7 @@ services:
     volumes:
      - "./docker/postgres.conf:/etc/postgresql/postgresql.conf"
     healthcheck:
-      test: [ "CMD", "pg_isready", "-h", "localhost" ]
+      test: pg_isready -U mmuser -d mattermost_test
```

Comment:
I think the culprit here is not using the correct env. variables. We should've been setting `PGUSER` and ` PGDATABASE` according to the https://www.postgresql.org/docs/current/libpq-envars.html. I think once we set the env vars correctly, the error should be ironed out @coltoneshaw